stl/CMakeLists.txt

```diff
@@ -552,20 +552,38 @@ function(add_stl_dlls D_SUFFIX THIS_CONFIG_DEFINITIONS THIS_CONFIG_COMPILE_OPTIO
     add_dependencies(msvcp${D_SUFFIX}_implib msvcp${D_SUFFIX} msvcp_1${D_SUFFIX} msvcp_2${D_SUFFIX} msvcp${D_SUFFIX}_atomic_wait msvcp${D_SUFFIX}_codecvt_ids)
     set_target_properties(msvcp${D_SUFFIX}_implib PROPERTIES STATIC_LIBRARY_OPTIONS "/NOLOGO;/NODEFAULTLIB;/IGNORE:4006;$<TARGET_LINKER_FILE:msvcp${D_SUFFIX}>;$<TARGET_LINKER_FILE:msvcp_1${D_SUFFIX}>;$<TARGET_LINKER_FILE:msvcp_2${D_SUFFIX}>;$<TARGET_LINKER_FILE:msvcp${D_SUFFIX}_codecvt_ids>;$<TARGET_LINKER_FILE:msvcp${D_SUFFIX}_atomic_wait>")
     set_target_properties(msvcp${D_SUFFIX}_implib PROPERTIES ARCHIVE_OUTPUT_NAME "msvcprt${D_SUFFIX}")
+
+    # HACK: for benchmarking, we need to export the standard library since we link google benchmark to the stl targets,

```

Comment:
We don't have any `HACK` comments today. I suggest:

* If this is an issue in a dependency (CMake?) that could be improved in the future, we should file an upstream issue and change this comment to say `TRANSITION`, which is how we mark hopefully-temporary workarounds.
* If this is just behavior that's inconvenient for us that we need to work around, forever, then we should just have the comment here but not mark it as `HACK`, since it is permanent.

Occurs below.